app/assets/scss/application.scss

```diff
@@ -62,23 +61,15 @@ $govuk-compatibility-govukelements: true;
 
 // Misc styles
 // TODO: Move misc styling into their own partial files or the Digital Marketplace FE Toolkit
-.padding-bottom-small {
-  padding-bottom: 10px;
-}
-
 .border-image {
   width: 100%;
-  border: 5px solid #dee0e2;
-}
-
-.lead {
-  margin-bottom: 20px;
+  border: govuk-spacing(1) solid #dee0e2;
 }
 
 .search-summary-border-bottom {
```

Comment:
Can we rename these styles to `.dmp-search-summary-border-bottom` or something like that?

Reply:
We can but I don't think now is the right time to do it because they might not be needed once we using govuk-frontend template. 

I have made a note to revisit the class names once we have migrated over.